_src/_entries/000009-entry-tufte-css-test.markdown

```diff
@@ -0,0 +1,17 @@
+---
+type:         post
+title:        "Tufte-CSS Test"
+date:         2016-03-14T10:00:00Z
+published:    false
+tags:
+  - pivotal
+  - work
+description: >
+  Tufte-CSS Test
+---
+
+Tufte CSS provides tools to style web articles using the ideas demonstrated by Edward Tufte’s books and handouts. Tufte’s style is known for its simplicity, extensive use of sidenotes, tight integration of graphics with text, and carefully chosen typography.
+
+# Side Notes
+
+One of the most distinctive features of Tufte’s style is his extensive use of sidenotes.{% include sidenote.html reference="sidenote-example" content="This is a sidenote." %} Sidenotes are like footnotes, except they don’t force the reader to jump their eye to the bottom of the page, but instead display off to the side in the margin. Perhaps you have noticed their use in this document already. You are very astute.
```

Comment:
How about if the reference is omitted, the CSS counter is used? Then you get the best of both worlds: no need to specify a reference if you don't want to; alternatively if you forget, nothing gets broken.